src/components/common/DomListener.tsx

```diff
@@ -4,6 +4,7 @@ import ReactDOM from "react-dom";
 // no tests because js-dom doesn't support MutationObserver
 // https://github.com/jsdom/jsdom/issues/639
 
+// Why do we differentiate `resize` from `domChange`? `domChange` can be anything liek att, childList, subtree & characterData also
```

Comment:
no specific reason as we listen to 2 different kind of event I thought would be nice to give back what's causing the function to be called

Reply:
Hmm im not sure if there's any value in this. Considering that you have to go all the way to curry the function without adding `curry` at the front/back. That's plain blasphemy.

Reply:
see #2